api/pb/api.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 package pb;
 
+option go_package="api/pb";
```

Comment:
This file, and all the included protobuf files, will be deprecated very soon - let's not bother updating them, okay?

Reply:
Ok, I didn't know that. The reason I added these line to get rid of the annoying warning from generating the proto files.